alerts/cloud.markdown

```diff
@@ -1,11 +0,0 @@
----
```

Comment:
Can this be removed now with `homeassistant_alerts` integration, or do we need to handle that differently?

Reply:
I think so.

Reply:
Inactive, non existing, alerts will be removed from issues.

https://github.com/home-assistant/core/blob/5eb3473d4191ffd04eeec3ef7f63c0adaea4afb3/homeassistant/components/homeassistant_alerts/__init__.py#L92-L94